internal/controller/controllers/agent_controller.go

```diff
@@ -562,6 +562,7 @@ func requirementsMet(agent *aiv1beta1.Agent, status string) {
 func (r *AgentReconciler) updateNtpSources(log logrus.FieldLogger, host *models.Host, agent *aiv1beta1.Agent) error {
 	if host.NtpSources == "" {
 		log.Debugf("Skip update NTP Sources: Host %s NTP sources not set", agent.Name)
+		log.Infof("Skip update NTP Sources: Host %s NTP sources not set", agent.Name)
```

Comment:
Do we need both lines?

Reply:
actually i added the second line to debug the subsystem failure, i will remove it